src/admin-views/attendees.php

```diff
@@ -90,16 +102,17 @@
 
 					<ul>
 						<?php
+						/** @var Tribe__Tickets__Ticket_Object $ticket */
 						foreach ( $tickets as $ticket ) {
-							/** @var Tribe__Tickets__Ticket_Object $ticket */
+							$ticket_name = sprintf( '%s [#%d]', $ticket->name, $ticket->ID );
 							?>
-						<li>
-							<strong><?php echo esc_html( $ticket->name ) ?>:&nbsp;</strong><?php
-							echo esc_html( tribe_tickets_get_ticket_stock_message( $ticket ) );
-						?></li>
-					<?php } ?>
+							<li>
+								<strong><?php echo esc_html( $ticket_name ) ?>:&nbsp;</strong><?php
```

Comment:
* :no_entry_sign: Inline PHP statement must end with a semicolon
* :no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: a8df65c9aef50c6b363cfda6103eb10f</sub>